### DISCUSSION TITLE
Implement one-time `get` for Firestore and Realtime Database

### DISCUSSION THREAD

**Discussion by jhuleatt** [FirebaseExtended/reactfire#137] [Ideas]
The Firebase JS SDK has a [`get`](https://firebase.google.com/docs/reference/js/firebase.firestore.Query#get) method for Firestore and Realtime Database that fetches a value once and doesn't stream updates.

We should probably implement something like a `useFirestoreDocOnce` to allow for this behavior in reactfire. Or, add a `once` flag to ReactFireOptions to accomplish the same thing.

- [x] `useFirestoreDoc[Data]Once` #211 
- [ ] `useFirestoreCollection[Data]Once`
- [ ] `useDatabaseObject[Data]Once`
- [ ] `useDatabaseList[Data]Once`

**Comment by forrest-keller** [comment]
I think we should use `getFirestoreDoc`, as used in the official Firebase APK
